src/metrics/migrations/0016_auto_20180205_0102.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
not sure, what the dubious purpose is? Required by one of the upgrades?

Reply:
the migration doesn't change anything, I don't know why Django is insisting on it.

Reply:
👍